lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -326,6 +327,8 @@ export class UserAgentApplication {
      * @param {@link (AuthenticationParameters:type)}
      */
     loginRedirect(userRequest?: AuthenticationParameters): void {
+        this.logger.verbose("LoginRedirect has been called");
```

Comment:
Should we include logging for `handleRedirectCallback` and/or hash processing in this PR as well?

Reply:
Good point. I wasn't sure how small to keep this PR. I will add logging for those too.

Reply:
@tnorling I added logging for `handleRedirectCallback`. However, there is a section in the code labelled "Response" that I will tackle in another PR that includes functions related to hash processing, so I will do it there.